### PR TITLE
Issue/#56 set up routes for backend - Work In Progress

### PR DESCRIPTION
**Pull request by jonathanvenema98**
- [ ] The pull request is complete according to the following criteria:
  - [ ] Acceptance criteria have been met
  - [ ] The documentation is kept up-to-date
  - [ ] Comprehensive tests (if applicable) have been generated and all pass.
  - [ ] The pull request describes the changes that have been made, and enough information is present in the description for any developer to understand what has changed
  - [ ] Commits have been squashed (or will be on merge).
  - [ ] The branch name is descriptive and follows the pull request title format : {issue/bug...}/(Issue Number) - Name of issue. E.g bug/30-Fix-Project
  - [ ] The pull request title is of the following format : {issue/bug...}/(Issue Number) - Name of issue. E.g bug/30-Fix-Project
  - [ ] At least two reviewers assigned. One of which must be the assigner of the issue.
  - [ ] If there are merge conflicts, run git rebase as opposed to git merge with master.

Description
- API implemented:
- User API - Get, Post
- Schedule API - Post
- Broken API -> Schedule - Get by "user", Delete by "taskId"

### CODE REVIEW

**Comment by Dinith1** [comment]
Looks like the build is failing due to some tests failing

**Comment by lucycy98** [comment]
@Dinith1  yeah its WIP, I just wanted to see how they were going

File: doto-backend/test/task.test.js [hunk 1]
```diff
@@ -11,6 +11,24 @@ const validUser = new UserModel({
     themePreference: 'dark'
 })
 
+const invalidUser = new UserModel({
```
Comment (tigerj2) [inline doto-backend/test/task.test.js]: invalidUser is not currently being used. It's probably better if we delete it for now.

File: doto-backend/src/models/Task.js [hunk 2]
```diff
@@ -1,15 +1,20 @@
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
+var uniqueValidator = require('mongoose-unique-validator');
 
 const taskSchema = mongoose.Schema({
     user: {
-        type: Schema.Types.ObjectId,
```
Comment (lucycy98) [inline doto-backend/src/models/Task.js]: was there a reason you changed this line?

**Pull request merged**